alby/alby_oauth_service.go

```diff
@@ -645,11 +646,38 @@ func (svc *albyOAuthService) ConsumeEvent(ctx context.Context, event *events.Eve
 		return
 	}
 
-	// TODO: we should have a whitelist rather than a blacklist, so new events are not automatically sent
+	// ensure we do not send unintended events to Alby API
+	eventWhitelist := []string{
```

Comment:
can we move this outside of the function into some constant?

Reply:
I moved it to a separate function (unfortunately in go there is no constant for arrays)